rpc/core/pipe.go

```diff
@@ -154,3 +154,12 @@ func validatePerPage(perPage int) int {
 	}
 	return perPage
 }
+
+func validateSkipCount(page, perPage int) int {
+	skipCount := (page - 1) * perPage
```

Comment:
I don't think this is correct. How skipCount can be negative? if page >= 1

Reply:
You should be right but I think validatePage can return a 0 - ie. `validatePage(1, 1, 0) == 0`

Reply:
I will double check.